test/none.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('none', function() {
+    var even = function(n) {return n % 2 === 0;};
+    var T = function() {return true;};
+
+    it('returns true if no elements satisfy the predicate', function() {
+        assert.strictEqual(R.none(even, [1, 3, 5, 7, 9, 11]), true);
+    });
+
+    it('returns false if any element satisfies the predicate', function() {
+        assert.strictEqual(R.none(even, [1, 3, 5, 7, 8, 11]), false);
+    });
+
+    it('returns true for an empty list', function() {
+        assert.strictEqual(R.none(T, []), true);
+    });
+
+    it('short-circuits on first false value', function() {
```

Comment:
I agree with @buzzdecafe that we should remove this test. We should leave the order in which we test the elements unspecified to avoid tying ourselves to a particular implementation.